austrakka/utils/options.py

```diff
@@ -202,7 +233,7 @@ def opt_organisation(**attrs: t.Any):
 
 def opt_group(**attrs: t.Any):
     defaults = {
-        'required': True,
+        'required': False,
```

Comment:
I think only the sequence commands are using this option, but given they're all passing in required=True, what's the motivation for changing the default to False?

I've also just noticed we have both opt_group and opt_group_name representing the same --group-name parameter, this was a preexisting problem by the looks of it! These should be consolidated. opt_group is currently multiple, but it doesn't need to be, as we ALSO have opt_groups which is multiple.

Reply:
I think I will go through this with you so I don't make a mistake when culling duplicates and changing misused options

Reply:
I think I resolved the duplicate issue with sam 👍